packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -78,11 +78,11 @@ _The key elements of the component's public API surface:_
 
 -   Component Name: `nimble-wafer-map`
 -   Props/Attrs:
```

Comment:
Make sure to capture any changes discussed in the api 👍

Reply:
Sure, I updated